package.json

```diff
@@ -41,7 +41,7 @@
 	"activationEvents": [
 		"onStartupFinished"
 	],
-	"main": "./out/src/extension.js",
+	"main": "./out/extension.js",
```

Comment:
I see it is because of tsconfig `rootDir` option, I am just wondering if it wold be better to do in another PR because this can be changed also in other repositories

Reply:
@djelinek By default, `src` was added due to the package.json import, after I updated the imports (where the package was used), the folder stopped being transferred even without this config. So I had to add ` "rootDir": "."` now